drakrun/drakrun/draksetup.py

```diff
@@ -475,6 +482,208 @@ def mount(iso_path, domain_name):
     subprocess.run(['xl', 'qemu-monitor-command', domain_name, f'change ide-5632 {iso_path_full}'])
 
 
+def get_minio_client(config):
+    minio_cfg = config['minio']
+    mc = Minio(endpoint=minio_cfg['address'],
+               access_key=minio_cfg['access_key'],
+               secret_key=minio_cfg['secret_key'],
+               secure=minio_cfg.getboolean('secure', fallback=True))
+    return mc
+
+
+def minio_prefix_empty(mc, bucket, prefix):
+    return len(list(mc.list_objects(bucket, prefix=prefix))) == 0
```

Comment:
This is used referenced only one in code and the logic is quite simple. Should this be a separate function?

Reply:
:+1